### PR TITLE
Porous Jump Python Interface

### PR DESCRIPTION
**Pull request by awccoppFC**
Added interface and tests for porous jump boundary condition

### CODE REVIEW

File: flow360/component/simulation/models/surface_models.py [hunk 1]
```diff
@@ -693,6 +695,57 @@ def ensure_surface_existence(cls, value):
...
+        + "viscous loss term. The value defines the coefficient for the axis normal "
+        + "to the surface."
+    )
+    forchheimer_coefficient: InverseLengthType = pd.Field(
+        description="Forchheimer coefficient of the porous media model which determines "
+        + "the scaling of the inertial loss term."
+    )
+    thickness: LengthType = pd.Field(
+        description="thickness of the thin porous media on the surface"
+        + "the scaling of the inertial loss term."
```
Comment (benflexcompute) [inline flow360/component/simulation/models/surface_models.py:737]:
```suggestion

```

File: flow360/component/simulation/models/surface_models.py [hunk 2]
```diff
@@ -693,6 +695,60 @@ def ensure_surface_existence(cls, value):
...
+        description="Forchheimer coefficient of the porous media model which determines "
+        + "the scaling of the inertial loss term."
+    )
+    thickness: LengthType = pd.Field(
+        description="Thickness of the thin porous media on the surface"
+    )
+
+    @pd.field_validator("entity_pairs", mode="after")
+    @classmethod
+    def ensure_surface_existence(cls, value):
```
Comment (benflexcompute) [inline flow360/component/simulation/models/surface_models.py:741]: Can you write a unit test for the validator. There are many examples in test_validators_params.py. You can just construct ad-hoc Surface.
